docker/Dockerfile

```diff
@@ -21,57 +21,57 @@ RUN apt-get -qq update && \
     rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
 # Install desired packages using pip
-# *not sure why, but when run in a single line the dependencies for stubs aren't installed
+# *not sure why, but when run in a single line the dependencies for smart aren't installed
 # (e.g. upgrade numpy from version 13.x to 16.x)
 RUN python3 -m pip install meshio
 RUN python3 -m pip install bpython
 RUN python3 -m pip install gmsh
-RUN python3 -m pip install fenics-stubs
+RUN python3 -m pip install fenics-smart
 #RUN python3 -m pip install \
 #    meshio \
 #    bpython \
 #    gmsh \
-#    fenics-stubs
+#    fenics-smart
 
 # bashrc settings (this appends to the top of the file)
 RUN printf \
-"# colored terminal output\n\
-export TERM=xterm-256color\n" | cat - $FENICS_HOME/.bashrc > temp && \
-mv temp $FENICS_HOME/.bashrc
+    "# colored terminal output\n\
+    export TERM=xterm-256color\n" | cat - $FENICS_HOME/.bashrc > temp && \
+    mv temp $FENICS_HOME/.bashrc
 
 # Bash aliases
 RUN printf \
-"# Our bash aliases\n\
-alias ..='cd ..'\n\
-alias ...='cd ../..'\n\
-alias ....='cd ../../..'\n\
-alias python='python3'\n\
-alias bpython='python3 -m bpython'\n\
-alias sourceme='source ~/.bashrc'\n\
-alias vimbash='vim ~/.bashrc'\n\
-alias vimalias='vim ~/.bash_aliases'\n\
-alias vimrc='vim ~/.vimrc'" >> $FENICS_HOME/.bash_aliases
+    "# Our bash aliases\n\
+    alias ..='cd ..'\n\
+    alias ...='cd ../..'\n\
+    alias ....='cd ../../..'\n\
+    alias python='python3'\n\
+    alias bpython='python3 -m bpython'\n\
+    alias sourceme='source ~/.bashrc'\n\
+    alias vimbash='vim ~/.bashrc'\n\
+    alias vimalias='vim ~/.bash_aliases'\n\
+    alias vimrc='vim ~/.vimrc'" >> $FENICS_HOME/.bash_aliases
 
 # vimrc settings
 RUN printf \
-"\" Our vimrc settings\n\
-syntax on\n\
-filetype plugin indent on\n\
-set tabstop=4\n\
-set shiftwidth=4\n\
-set expandtab\n\
-set hlsearch\n\
-colorscheme desert\n\
-inoremap kj <Esc>\n\
-vnoremap x \"_x\n\
-nnoremap x \"_x\n\
-vnoremap c \"_c\n\
-nnoremap c \"_c\n\
-set visualbell\n\
-set t_vb=\n\
-set number\n\
-set colorcolumn=120\n\
-highlight ColorColumn ctermbg=7" >> $FENICS_HOME/.vimrc
+    "\" Our vimrc settings\n\
+    syntax on\n\
+    filetype plugin indent on\n\
+    set tabstop=4\n\
+    set shiftwidth=4\n\
+    set expandtab\n\
+    set hlsearch\n\
+    colorscheme desert\n\
+    inoremap kj <Esc>\n\
+    vnoremap x \"_x\n\
+    nnoremap x \"_x\n\
+    vnoremap c \"_c\n\
+    nnoremap c \"_c\n\
+    set visualbell\n\
+    set t_vb=\n\
+    set number\n\
+    set colorcolumn=120\n\
+    highlight ColorColumn ctermbg=7" >> $FENICS_HOME/.vimrc
```

Comment:
For the docker file in the repo, should we just remove the aliases and vimrc settings, since that should probably just be determined by the user?